opacus/privacy_engine.py

```diff
@@ -276,6 +249,7 @@ def make_private_with_epsilon(
             module=module,
             optimizer=optimizer,
             data_loader=data_loader,
+            # TODO: what if the client has selected non-RDP accountant?
```

Comment:
This can be handled if accountant is not fixed during construction and instead passed here as arg.

Reply:
The problem here is that we don't yet have reverse epsilon computation for Gaussian DP (i.e. method to get noise multiplier given the target epsilon)